example/lib/main.dart

```diff
@@ -17,39 +25,32 @@ class _HomePageState extends State<HomePage> {
   @override
   void initState() {
     super.initState();
-    // Can't show a dialog in initState, delaying initialization
-    WidgetsBinding.instance!.addPostFrameCallback((_) => initPlugin());
+    initPlugin();
```

Comment:
`WidgetsBinding.instance!.addPostFrameCallback` is not necessary, awaiting `AppTrackingTransparency.trackingAuthorizationStatus` already delays `showCustomTrackingDialog`.